drift/info_parser.py

```diff
@@ -196,6 +194,10 @@ def _find_group(name, grouped_comps):
         else:
             grouped_comparisons.append(comparison)
 
+    # Need to add all dot notation comparisons into the output grouped_comparisons list
+    for comparison in grouped_comparisons_with_dotnotation:
+        grouped_comparisons.append(grouped_comparisons_with_dotnotation.get(comparison))
```

Comment:
Is there a way to append it without a `for` loop? This adds another cycles.

Reply:
Fixed here: a4285a58dac2122380078195dd6d94d8173f9366

Reply:
Please fix it up with commit "Using dict instead of list". The issue was introduced in this PR so it does not need to be carried over to main branch.